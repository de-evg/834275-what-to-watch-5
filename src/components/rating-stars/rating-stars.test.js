import React from "react";
import renderer from "react-test-renderer";
import RatingStars from "./rating-stars";


it(`Should RatingStars unactive render correctly`, () => {
  const tree = renderer
    .create(<RatingStars
      activeStar={`star-0`}
      onRatingInputChange={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
