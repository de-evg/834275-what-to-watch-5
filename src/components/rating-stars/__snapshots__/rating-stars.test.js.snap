// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should RatingStars unactive render correctly 1`] = `
<div
  className="rating__stars"
>
  <input
    checked={false}
    className="rating__input"
    id="star-0"
    name="rating"
    onChange={[Function]}
    required={true}
    type="radio"
    value={1}
  />
  <label
    className="rating__label"
    htmlFor="star-0"
  >
    Rating 
    1
  </label>
  <input
    checked={false}
    className="rating__input"
    id="star-1"
    name="rating"
    onChange={[Function]}
    required={true}
    type="radio"
    value={2}
  />
  <label
    className="rating__label"
    htmlFor="star-1"
  >
    Rating 
    2
  </label>
  <input
    checked={false}
    className="rating__input"
    id="star-2"
    name="rating"
    onChange={[Function]}
    required={true}
    type="radio"
    value={3}
  />
  <label
    className="rating__label"
    htmlFor="star-2"
  >
    Rating 
    3
  </label>
  <input
    checked={false}
    className="rating__input"
    id="star-3"
    name="rating"
    onChange={[Function]}
    required={true}
    type="radio"
    value={4}
  />
  <label
    className="rating__label"
    htmlFor="star-3"
  >
    Rating 
    4
  </label>
  <input
    checked={false}
    className="rating__input"
    id="star-4"
    name="rating"
    onChange={[Function]}
    required={true}
    type="radio"
    value={5}
  />
  <label
    className="rating__label"
    htmlFor="star-4"
  >
    Rating 
    5
  </label>
</div>
`;
